js/map.js

```diff
@@ -0,0 +1,96 @@
+import { createCard } from './card.js';
+import { enablePage } from './page-state.js';
+
+const DEFAULT_COORDS = {
+  lat: 35.67500,
+  lng: 139.75000,
+};
```

Comment:
Здесь лучше подойдет синтаксис перечисления
```suggestion
const DefaultCoords = {
  LAT: 35.67500,
  LNG: 139.75000,
};
```